pyproject.toml

```diff
@@ -57,6 +57,7 @@ ignore = [
     "B011",  # Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`
     "E731",  # Do not assign a `lambda` expression, use a `def`
     "PT011", # `pytest.raises(OSError)` is too broad
+    "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
```

Comment:
This one is tricky.  I fixed a few of these in systemd_ctypes yesterday as well.

In general it's usually possible to change lists to tuples to avoid this, with a minor reduction in clarity.  I'm not a big fan of the fact that this also gets complained about for class variables which *are* defined in the superclass with `ClassVar` and then just simply assigned in the subclass... but that's sort of a result of ruff's status as a simple linter...

In general this warning seems like a better fit for something like `mypy --strict`.

Which is all to say: there's enough discussion to be had about this particular issue that I completely and totally agree with your decision to ignore it now and deal with it later...